modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -83,15 +81,18 @@ In your network, you might need to whitelist the hostnames and ports of external
 [[install_Istio]]
 == Download, Install, and Configure Istio
 
-You can install Istio 1.6.x (`1.6.0` to `1.6.8`) or 1.7.x in your environment to support Anypoint Service Mesh.
+Anypoint Service Mesh supports Istio versions 1.6.0 through 1.6.8 and 1.7.0. You can install any of these Istio versions in your environment.
```

Comment:
"Anypoint Service Mesh supports Istio versions 1.6.0 through 1.6.8 and 1.7.0."
I would change that to "Anypoint Service Mesh supports Istio versions 1.6.0 through 1.6.8 and version 1.7.0."

Reply:
Done! Thanks!